### PR TITLE
✨ (api): Added request for access and access status scaffolding

### PR DESCRIPTION
**Pull request by FaraiQC**
(packages): added basic scripts to the services folder of access-status

### CODE REVIEW

File: changelog.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
+<li>%h %ci %d %s</li>
```
Comment (arulagrawal) [inline changelog.html:17]: I'm not too familiar with what scripts you're adding. Why is there a changeling.html in the root of the project with these contents?

File: yarn.lock [hunk 2]
```diff
@@ -4111,15 +4111,6 @@ apollo-server-types@^3.5.2:
...
 
-apollo-server@^3.6.5:
-  version "3.6.5"
-  resolved "https://registry.yarnpkg.com/apollo-server/-/apollo-server-3.6.5.tgz#74b488f32c7039a2b96af0200dd41277f1cbec8e"
-  integrity sha512-IE3tdR7ai57hKw4APoTeVrR7K9cDjgznMlMmWil6VrRPlWMg5lMFumatDJL1ndCfV9zHbykAnaL+N9RycIMMWw==
-  dependencies:
-    apollo-server-core "^3.6.5"
-    apollo-server-express "^3.6.5"
-    express "^4.17.1"
-
```
Comment (arulagrawal) [inline yarn.lock:4122]: Why are we messing with the lock file? I think we shouldn't be doing this. @arneschreuder

File: yarn.lock [hunk 3]
```diff
@@ -2471,7 +2471,7 @@
     v8-compile-cache "2.3.0"
     yargs-parser "20.0.0"
 
-"@nrwl/cli@13.9.4", "@nrwl/cli@^13.9.4":
```
Comment (arulagrawal) [inline yarn.lock:2474]: Not sure what's going on here, but see other comment on this file

File: doc/doc-filelist.js [hunk 4]
```diff
@@ -0,0 +1 @@
+var tree={};
```
Comment (arulagrawal) [inline doc/doc-filelist.js:1]: Judging by the PR message, this doc folder and its files aren't meant to be in this PR?

File: libs/api/access-status/shared/feature/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# api-access-status-shared-feature
```
Comment (arulagrawal) [inline libs/api/access-status/shared/feature/README.md:1]: I'm not sure about the /shared path of these files. What is shared/feature doing?

**Review by arulagrawal — CHANGES_REQUESTED** [review]
There are some things I'm unsure about, please clarify these if you can.
This PR is pretty big. It has changes to /doc /libs/api/access-status and /lib/api/request-access. Like I said in one of the comments I'm not sure the /doc part fits in this PR?

@arneschreuder should you please look at this PR?

**Comment by arulagrawal** [comment]
These changes aren't 📦️ packages, please update the PR title and commit messages.

File: libs/api/access-status/shared/feature/README.md (hunk 5 above)
Comment (FaraiQC) [reply libs/api/access-status/shared/feature/README.md:1]: The /shared path should not be there. I only notice now that I put it there.

File: changelog.html (hunk 1 above)
Comment (FaraiQC) [reply changelog.html:17]: I ran this command "nx g @nrwl/nest:lib api/access-status/service/feature"  on the terminal in order to generate the services subfolder of my feature. The folder structure generated by this command is the same as that made by other service engineers.

Comment (arulagrawal) [reply changelog.html:17]: Okay but what does this have to do with a changelog.html with gibberish in the root?

**Comment by arulagrawal** [comment]
Could you please expand the PR description? Basic scripts to do what? What do you mean scripts? What feature are you adding? What are we adding to access-status in this PR?

File: yarn.lock (hunk 2 above)
Comment (FaraiQC) [reply yarn.lock:4122]: I am not sure why there are changes made to this file. Could it have something to do with this command that I ran on the terminal "nx g @nrwl/nest:lib api/access-status/service/feature"?

File: yarn.lock (hunk 3 above)
Comment (FaraiQC) [reply yarn.lock:2474]: I am not sure why there are changes made to this file. Could it have something to do with this command that I ran on the terminal "nx g @nrwl/nest:lib api/access-status/service/feature"?

**Comment by FaraiQC** [comment]
> Could you please expand the PR description? Basic scripts to do what? What do you mean scripts? What feature are you adding? What are we adding to access-status in this PR?

These scripts were added by me using this command "nx g @nrwl/nest:lib api/access-status/service/feature" in order to match the example folder structure given to us. What do you suggest I name this pull request message (instead of "[(packages): added basic scripts]")

**Comment by FaraiQC** [comment]
I closed the PR because some tests started failing when I removed the shared folder (that should not have been there)

File: libs/api/access-status/shared/feature/README.md (hunk 5 above)
Comment (FaraiQC) [reply libs/api/access-status/shared/feature/README.md]: I removed the shared folder

File: changelog.html (hunk 1 above)
Comment (arneschreuder) [reply changelog.html:17]: I agree with arulagrawal. Also, you probably want to use the nx console plugin in VSCode to generate these. The command line works fine, but the parameters are a bit uncertain in you use the cli.

File: doc/doc-script.js [hunk 6]
```diff
@@ -0,0 +1,228 @@
+// # res/script.js
```
[the remark this answers is not in the document]
Comment (FaraiQC) [reply doc/doc-script.js:1]: let me remove the doc folder

**Review by FaraiQC — COMMENTED** [review]
I removed the files that should not have been in the PR (the doc folder).

**Comment by arneschreuder** [comment]
Tests pass. Resolve all conversations and then you can merge

File: changelog.html (hunk 1 above)
Comment (arulagrawal) [reply changelog.html:17]: This is not resolved, this file is still present

File: yarn.lock (hunk 2 above)
Comment (arulagrawal) [reply yarn.lock:4122]: I think we should still try to leave this alone. As per @arneschreuder's comment, no package.json is strange

**Review by arulagrawal — CHANGES_REQUESTED** [review]
Still need to remove changelog and agree on yarn.lock

File: changelog.html (hunk 1 above)
Comment (FaraiQC) [reply changelog.html]: I just removed the changelog file as requested

**Review by arneschreuder — APPROVED** [review]
LGTM

**Review by arulagrawal — APPROVED** [review]

**Comment by arulagrawal** [comment]
🚀

**Pull request merged**